content/rs/references/crdb-cli-reference.md

```diff
@@ -0,0 +1,284 @@
+|`encryption true`| boolean| Enable encryption|
+|`sharding disable`| string| Disable sharding (also known as [database clustering]({{< relref "/rs/concepts/high-availability/replication.md" >}})) so that there is only one shard for the database|
+|`shards-count <number>`| integer| If sharding is enabled this specifies the number of Redis shards for each database instance|
+|`shard-key-regex <regex_rule>`| string| If sharding is enabled this defines a custom sharding key regex rule (also known as a [hashing policy]({{< relref "/rs/concepts/high-availability/clustering#custom-hashing-policy" >}}) that determines which keys are located in each shard|
+<!-- |`default-db-config <options>`|text|Default database configuration options|
+|`default-db-config-file <filename>`|file path|Default database configuration options| -->
+
+#### crdb create examples
+
+To create an Active-Active database with two shards in each instance and with encrypted traffic between the clusters:
+
+```sh
+crdb-cli crdb create --name mycrdb --memory-size 100mb --port 12000 --instance fqdn=cluster1.local,username=test,password=test --instance fqdn=cluster2.local,username=test,password=test --shards-count 2 --encrypted true
+```
+
+To create an Active-Active database with no shards and with encrypted traffic between the clusters:
+
+```sh
+crdb-cli crdb create --name mycrdb --memory-size 100mb --port 12000 --instance fqdn=cluster1.local,username=test,password=test --instance fqdn=cluster2.local,username=test,password=test --encrypted true --shards-count 2
+```
+
+To create an Active-Active database with 1 shard in each instance and not wait for the response:
+
+```sh
+crdb-cli crdb create --name mycrdb --memory-size 100mb --port 12000 --instance fqdn=cluster1.local,username=test,password=test --instance fqdn=cluster2.local,username=test,password=test --no-wait
+```
+
+### Showing the list of Active-Active Databases
+
+The `crdb list` command shows all Active-Active databases that the cluster participates in.
+Each database is represented with a unique GUID, the name of the database, an instance ID, and the FQDN of the cluster that hosts the instance.
+You use the CRDB-GUID and REPL-ID in other commands, such as [add-instance](#adding-an-instance-to-an-activeactive-database) and [remove-instance](#removing-an-instance-from-an-activeactive-database).
+
+```sh
+crdb-cli crdb list
+```
+
+For example:
+
+```sh
+# crdb-cli crdb list
+CRDB-GUID                                              NAME    REPL-ID     CLUSTER-FQDN
+e7ef4552-3705-4790-9f30-6a8e9f4bef28  shopping-cart  1        cluster1.local
+e7ef4552-3705-4790-9f30-6a8e9f4bef28  shopping-cart  2        cluster2.local
+```
+
+### Updating the configuration of an Active-Active database
+
+The `crdb update` command lets you change the configuration of an existing Active-Active database.
```

Comment:
I'm missing here:
- crdb update is a Global configuration change which will affect all instances.
- A user can also update local configurations via rladmin, users should take this into account.